octodns_dnsmadeeasy/__init__.py

```diff
@@ -236,7 +237,10 @@ def _data_for_CAA(self, _type, records):
         return {'ttl': records[0]['ttl'], 'type': _type, 'values': values}
 
     def _data_for_TXT(self, _type, records):
-        values = [value['value'].replace(';', '\\;') for value in records]
+        values = [
+            re.sub(r'(?<!\\)\"\"', '', value['value'].replace(';', '\\;'))
```

Comment:
This regex is involved enough that it could probably use a comment about what it's doing/how it's doing it. 

Less important, but might be worth pulling it out into a class static variable and referenced it here with `self.VARIABLE_NAME` so that it's not recompiled for every TXT value.

Reply:
Changes made as suggested.